package.json

```diff
@@ -18,14 +18,28 @@
     "babel-preset-env": "~1.6.0",
     "clean-webpack-plugin": "~0.1.17",
     "css-loader": "~0.28.7",
+    "eslint": "^4.16.0",
```

Comment:
All of these linting packages aren't really necessary. Feel free to remove what you like.